apps/jobs/constants/values.py

```diff
@@ -32,3 +29,10 @@
 DB_TABLE_COMPANY = "tbl_company"
 DB_TABLE_JOBS = "tbl_job"
 DB_TABLE_USER_PROFILE = "tbl_user_profile"
+
+RESUME_DOCUMENT_TYPE = "resume"
+PROFILE_PICTURE_DOCUMENT_TYPE = "profile_picture"
+COVER_LETTER_DOCUMENT_TYPE = "cover_letter"
+
+ITEMS_PER_PAGE = 5
+PAST_3_WEEK_DATETIME_DAYS18 = 18
```

Comment:
change the variable name to something else, something that resembles to the functionality for which we are using it.

Reply:
lets not changes the name they are as we discussed and as per the functionality.